python/tvm/relay/transform.py

```diff
@@ -39,19 +35,19 @@ class PassInfo(RelayNode):
 
     Parameters
     ----------
-    name : str
-        The pass name.
-
     opt_level : int
         The optimization level of this pass.
 
+    name : str
+        The pass name.
+
     required : List[str]
         The list of passes that are required by a certain pass.
     """
 
-    def __init__(self, name, opt_level, required=None):
-        self.__init_handle_by_constructor__(_transform.PassInfo, name, opt_level,
-                                            required)
+    def __init__(self, opt_level, name, required=None):
```

Comment:
Why should we put opt_level first? I don't even think most passes should be on a standard opt-level, we end up in the current system we have where people arbitrarily assign pass numbers to passes and when passes are non-robust we just bump them to a higher pass number.

Reply:
We should focus on names and enabling passes by name in most cases imo.

Reply:
I made the change just to be consistent with the rest of the positional argument, we can debate whether it is a good idea to put name or opt_level first. I am fine either way